src/Components/Facility/BedManagement.tsx

```diff
@@ -60,10 +60,15 @@ const BedRow = (props: BedRowProps) => {
 
   const handleDeleteConfirm = async () => {
     const res = await dispatchAction(deleteFacilityBed(id));
-    if (res && res.status == 204) {
+    if (res && res.status === 204) {
```

Comment:
Use optional chaining `res?.status`
```suggestion
    if (res?.status === 204) {
```